src/behaviors/dragPanBehavior.ts

```diff
@@ -0,0 +1,89 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Behavior {
+  export class DragPan {
+
+    private _dragInteraction: Interaction.Drag;
+    private _scale: Scale.AbstractQuantitative<any>;
+    private _leftBounds = [-Infinity, Infinity];
+    private _rightBounds = [-Infinity, Infinity];
+    private _verticalPan: boolean;
+
+    /**
+     * Creates a DragPan Behavior.
+     *
+     * The allows you to move around and zoom in on a plot, interactively. It
+     * does so by changing the xScale and yScales' domains repeatedly.
+     *
+     * @constructor
+     * @param {QuantitativeScale} [xScale] The X scale to update on panning/zooming.
+     * @param {QuantitativeScale} [yScale] The Y scale to update on panning/zooming.
```

Comment:
The doc is incorrect.  I considered having an xScale/yScale but then all of the below API endpoints would have to incorporate the addition of other scales, which is why I went for the simpler single scale constructor appraoch.